Revise docs for Auto-packets and auto-components (#2006)
o3de/o3de.org: content/docs/user-guide/networking/autopackets.md

### Include

> AzAutoGen uses the `Include` tag to generate the `#includes` of the C++ code. Use an `Include` tag for each header that your generated classes will use

Reviewer: ```suggestion
AzAutoGen uses the `Include` tag to generate the `#includes` of the C++ code. Use an `Include` tag for each header used by the generated classes.
```
